openedx/features/partners/constants.py

```diff
@@ -51,3 +51,13 @@
 
 PARTNER_USER_STATUS_WAITING = 'waiting'
 PARTNER_USER_STATUS_APPROVED = 'approved'
+
+PARTNERS_TOP_REGISTRATION_COUNTRIES = {
+    'give2asia': (
```

Comment:
Actually this is a confusion in ticket AC, these countries are not specific to give2asia or any partner these are for the platform, so let's please create a contants.py file in openedx/features and move the tuple to that file with appropriate name.

Reply:
As you mentioned, we are not sure about whether these are specific to partners or give2asia, we should be confirming this with the product team first.